quickannotator/db/crud/annotation.py

```diff
@@ -215,6 +219,26 @@ def create_annotation_table(image_id: int, annotation_class_id: int, is_gt: bool
         Base.metadata.create_all(bind=db_session.bind, tables=[table])
 
         return create_dynamic_model(table_name)
+        build_annotation_table_name
+    
+    @staticmethod
+    def delele_annotation_table(image_id: int):
```

Comment:
1. typo in method name
2. this method should only drop one table at a time. 

I would recommend making this a non-static method. it should be used by instantiating an AnnotationStore and then calling store.remove() to remove it.

Then if you want to loop over multiple tables, a second method should be declared outside of AnnotationStore.

Reply:
I changed the name to delete_annotation_tables_by_image_id. I want to keep it static since removing all annotation tables only requires knowing image id. We can have a method to only drop one table at a time based on the condition.